cmake/build_mingw32.sh

```diff
@@ -8,7 +8,7 @@ else
 	CMAKE_OPTS="$CMAKE_OPTS -DLMMS_BUILD_MSYS=1"
 fi
 
-export PATH=$PATH:$MINGW/bin
+export PATH=$MINGW/bin:$PATH
```

Comment:
> this might actually be the cause of #2951, I'll check if this change fixes that.

It shouldn't help that problem.  This should only fix problems where the same library exists in both `$PATH` as well as `$MINGW/bin` which I wouldn't expect to solve #2951.  This is just a change which fixes the order of resolution.  #2951 is more likely related to something with `pkg-config` or `CMAKE_PREFIX_PATH` since the library isn't found at all.